diff --git a/lib/index.js b/lib/index.js
index 994be10..f458c83 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -43,7 +43,7 @@ function generateXML (data){
     }
     channel.push({ generator:       data.generator });
-    channel.push({ lastBuildDate:   new Date().toUTCString() });
+    ifTruePush(data.lastBuildDate !== false, channel, { lastBuildDate: new Date(data.lastBuildDate || undefined).toUTCString() });

     ifTruePush(data.feed_url, channel, { 'atom:link': { _attr: { href: data.feed_url, rel: 'self', type: 'application/rss+xml' } } });
